src/egon/data/datasets/gas_neighbours/eGon2035.py

```diff
@@ -1498,17 +1516,19 @@ def calculate_ocgt_capacities():
     df_ocgt = df_ocgt.groupby(by=["bus0", "bus1"], as_index=False).sum()
 
     return df_ocgt
-
+node

```

Comment:
That tiny `node` was what broke the tests. :) Fixed in 8146d92099d0aacedb637d62de2a48f3760e1665. ;)

Reply:
Thanks for fixing :)